testing/fake-dom.js

```diff
@@ -185,41 +189,44 @@ export class FakeWindow {
     /** @const */
     this.Date = window.Date;
 
+    this.setTimeout = function() {
```

Comment:
The JSDoc below doesn't seem useful when switching the order like this. We can either add the missing params (prefixed with `unused`) or remove the JSDoc outright. I think it's fine to remove since they're just standard window APIs.

Ditto below.

Reply:
done. Thanks